app/serializers/host_serializer.rb

```diff
@@ -4,5 +4,12 @@
 class HostSerializer
   include FastJsonapi::ObjectSerializer
   set_type :host
-  attributes :name, :profiles, :compliant
+  attributes :name, :os_major_version, :os_minor_version, :last_scanned,
+             :rules_passed, :rules_failed
+  attribute :compliant do |host|
+    host.compliant.values.all?
+  end
+
+  has_many :test_results
+  has_many :profiles
```

Comment:
My intention here is to try and match the info returned by the related graphql system type.

Reply:
Is there maybe a way that we can abstract the definitions to use in both and keep their responses in sync?

Reply:
It's a good goal, but I'm not sure how we'd achieve it. For example, graphql allows fields with args, something we can't do with REST. Also, I'm not sure we want to lock graphql to REST because the UI depends on graphql - it's already useful and may be useful in the future to have graphql information available to the UI in some format the doesn't make sense in the REST API.